server/services/tp-link/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "gladys-tplink",
+  "version": "1.0.0",
+  "main": "index.js",
+  "os": [
+    "darwin",
+    "linux",
+    "win32"
+  ],
+  "cpu": [
+    "x64",
+    "arm",
+    "arm64"
+  ],
+  "dependencies": {
+    "bluebird": "^3.7.2",
```

Comment:
Relative to timer comment, if timer is requested, you can use bluebird Promise.delay().
But I can't see any usage.

Reply:
Fixed